UserStory.md

```diff
@@ -0,0 +1,12 @@
+# User Story 
+
+```
+As a user,
+I want to be able to categorize and search through my handwritten notes so that I can find the information that I need.
+```
+## Acceptance Criteria 
+- Sign Up and login : The user must be able to sign up and login into the app with the help of an email or other authentication methods
+- Uploading images: The user must be able to upload the images into the app using the camera or the images that are already in the device.
+- Tags generation: The user must receive revelant tags based on the handwritten documents automatically.
+- Tags Customization: The user must be able to manually add, edit or delete the tags for better customization. 
+- Searchability: The user must be able to search the documents based on the tags to locate specific documents.  
```

Comment:
Since the user story is in first person, the acceptance criteria should be as if the user who created the user story wants the app to do.
Eg: Instead of "The user must be able to...." write something like "I can sign up and login...." or "I should be able to signup and login...." ( whichever sounds perfect)